libbeat/outputs/codec/json/json.go

```diff
@@ -54,19 +57,23 @@ func init() {
 			}
 		}
 
-		return New(config.Pretty, info.Version), nil
+		return New(config.Pretty, config.EscapeHTML, info.Version), nil
 	})
 }
 
 // New creates a new json Encoder.
-func New(pretty bool, version string) *Encoder {
-	e := &Encoder{pretty: pretty, version: version}
+func New(pretty, escapeHTML bool, version string) *Encoder {
+	e := &Encoder{version: version, config: config{
+		Pretty:     pretty,
+		EscapeHTML: escapeHTML,
+	}}
 	e.reset()
 	return e
 }
```

Comment:
Maybe a later refactoring would change the signature to take an object instead. No need to do it now.